__tests__/app.test.js

```diff
@@ -113,3 +113,46 @@ describe('GET /api/articles', () => {
     })
 })
 
+describe('GET /api/articles/:article_id/comments', () =>{
+    test('200: returns an array of comment objects for the given article_id', () => {
+        return request(app).get('/api/articles/1/comments').expect(200).then(({ body }) => {
+            const { comments } = body
+            expect(comments).toHaveLength(11)
+            comments.forEach((comment) => {
+                expect(comment.article_id).toBe(1)
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                })
```

Comment:
Really good test, checking for a specific article ID while the rest of the properties can vary